app/v1/controllers/AmendLossClaimsOrderController.scala

```diff
@@ -77,6 +77,7 @@ class AmendLossClaimsOrderController @Inject()(val authService: EnrolmentsAuthSe
       case BadRequestError
            | NinoFormatError
            | TaxYearFormatError
+           | ClaimIdFormatError
```

Comment:
What is this error from? It's not in the spec as far as I'm aware as we don't have a ClaimId